### PR TITLE
adding interactive node search

### PR DESCRIPTION
**Pull request by lagillenwater**

### CODE REVIEW

**Review by bsantan — APPROVED** [review]

**Review by bsantan — CHANGES_REQUESTED** [review]
in DS_example_input.csv may need to use a different delimiter ("|" is what I have been using, we should standardize)

File: assign_nodes.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	examples = pd.read_csv(user_example_file)
+	return(examples)
+
+# Map the nodes to a dictionary
+def map_nodes(examples):
+	nodes = pd.melt(examples)
+	return(nodes)
+
+# Search through graph to find nodes based on input features
+def findNode(node_name, nodes, ontology = ""):
```
Comment (bsantan) [inline assign_nodes.py:14]: function names should be word1_word2 format

Comment (lagillenwater) [reply assign_nodes.py:14]: test

**Comment by lagillenwater** [comment]
Updated files with the requested changes

**Review by bsantan — APPROVED** [review]

**Review by Nourah-Salem — APPROVED** [review]

**Review by Nourah-Salem — APPROVED** [review]

**Comment by Nourah-Salem** [comment]
approval trial

**Review by Nourah-Salem — APPROVED** [review]

**Review by Nourah-Salem — APPROVED** [review]

**Review by Nourah-Salem — APPROVED** [review]

**Review by Nourah-Salem — APPROVED** [review]
reviewed

**Pull request merged**